app/uk/gov/hmrc/gform/views/govuk_wrapper.scala.html

```diff
@@ -204,7 +204,16 @@
   }
 }
 
-@getHelpForm = @{uiHelpers.reportAProblemLink(appConfig.reportAProblemPartialUrl, appConfig.reportAProblemNonJSUrl)}
+@getHelpForm = {
+        @if(templateId.value == "apply-customs-comprehensive-guarantee") {
+        <div class="report-error">
+            <a  href="/contact/accessibility?service=gForm&userAction=%2F@{templateId.value}%2F@title.replace(' ', '-')" target="_blank">@Messages("deskpro.new.content")</a>
```

Comment:
As explained yesterday, we do not want escaped html (`%2F` for instance) in our code, it is hard to read. Also `title.replace(' ', '-')` doesn't solve the problem of other special characters which can appear in `title`.